Buildings/Fluid/HeatExchangers/DXCoils/WaterCooled/Examples/VariableSpeed.mo

```diff
@@ -0,0 +1,120 @@
+within Buildings.Fluid.HeatExchangers.DXCoils.WaterCooled.Examples;
+model VariableSpeed "Test model for variable speed DX coil"
+  import Buildings;
+  package MediumAir = Buildings.Media.Air;
+  package MediumWater = Buildings.Media.Water;
+  extends Modelica.Icons.Example;
+ parameter Modelica.SIunits.MassFlowRate m_flow_nominal = datCoi.sta[datCoi.nSta].nomVal.m_flow_nominal
+    "Nominal mass flow rate";
+ parameter Modelica.SIunits.PressureDifference dpEva_nominal = 1000
+    "Pressure drop at m_flow_nominal";
+ parameter Modelica.SIunits.PressureDifference dpCon_nominal = 40000
+    "Pressure drop at mCon_flow_nominal";
+
+  Buildings.Fluid.Sources.Boundary_pT sinAir(
+    redeclare package Medium = MediumAir,
+    nPorts=1,
+    p(displayUnit="Pa"))
+              "Sink on air side"
+    annotation (Placement(transformation(extent={{52,30},{32,50}})));
+  Buildings.Fluid.Sources.MassFlowSource_T souAir(
+    redeclare package Medium = MediumAir,
+    use_T_in=true,
+    nPorts=1,
+    m_flow=1.5,
+    T=299.85) "Source on air side"
+    annotation (Placement(transformation(extent={{-48,28},{-28,48}})));
+  Modelica.Blocks.Sources.Ramp TEvaIn(
+    duration=600,
+    startTime=2400,
+    height=-5,
+    offset=273.15 + 23) "Temperature"
+    annotation (Placement(transformation(extent={{-100,40},{-80,60}})));
+  Buildings.Fluid.HeatExchangers.DXCoils.WaterCooled.VariableSpeed varSpeDX(
+    redeclare package Medium1 = MediumAir,
+    redeclare package Medium2 = MediumWater,
+    dpEva_nominal=dpEva_nominal,
+    datCoi=datCoi,
+    show_T=true,
+    dpCon_nominal=dpCon_nominal,
+    minSpeRat=datCoi.minSpeRat)  "Variable speed DX coil"
+    annotation (Placement(transformation(extent={{-6,-6},{14,14}})));
+
+  Buildings.Fluid.HeatExchangers.DXCoils.WaterCooled.Data.Generic.DXCoil datCoi(nSta=1, sta={
+        Buildings.Fluid.HeatExchangers.DXCoils.WaterCooled.Data.Generic.BaseClasses.Stage(
+        spe=1800/60,
+        nomVal=WaterCooled.Data.Generic.BaseClasses.NominalValues(
+          Q_flow_nominal=-21000,
+          COP_nominal=3,
+          SHR_nominal=0.8,
+          m_flow_nominal=1.5,
+          mCon_flow_nominal=1,
+          TEvaIn_nominal=273.15+26.67,
+          TConIn_nominal=273.15+29.4),
+        perCur=Buildings.Fluid.HeatExchangers.DXCoils.WaterCooled.Examples.PerformanceCurves.Curve_I())}) "Coil data"
+    annotation (Placement(transformation(extent={{60,60},{80,80}})));
+  Modelica.Blocks.Sources.Ramp     mCon_flow(
+    duration=600,
+    startTime=6000,
+    height=0,
+    offset=1)                                     "Condensor inlet mass flow"
```

Comment:
Correct formatting in this file (and maybe in other files as well that have superfluous empty spaces or line breaks.